examples/test/azureml/test_bert_ptq_cpu_aml.py

```diff
@@ -24,8 +25,6 @@ def setup():
     [
         # aml system test
         ("bert_ptq_cpu.json", "tpe", "joint", "aml_system"),
-        # aml model test in local system
-        ("bert_ptq_cpu_aml.json", False, None, "local_system"),
```

Comment:
I think this test case is still useful. Helps us make sure aml model paths can also be used for local runs.

Reply:
Oh yeah that's true. I thought this will be tested in aml system as well, while aml system doesn't download aml models to local. I'll add this back.

Reply:
Added back